### ISSUE TITLE
Bleadperl breaks SALVA/Sort-Packed-0.07.tar.gz

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#13869]
Migrated from [rt.perl.org#121944](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=121944) (status was 'resolved')

Searchable as RT121944$

**Comment by p5pRT** [comment]
### From @chorny

threaded perls only\, any OS

Packed\.xs&#8203;: In function ‘shuffle\_packed’&#8203;:
Packed\.xs&#8203;:216&#8203;:32&#8203;: error&#8203;: ‘my\_perl’ undeclared \(first use in this function\)
Packed\.xs&#8203;:216&#8203;:32&#8203;: note&#8203;: each undeclared identifier is reported only once
for each function it appears in

http&#8203;://www\.cpantesters\.org/cpan/report/d0d3fec2\-de62\-11e3\-be5c\-be9c6dfca90a
http&#8203;://www\.cpantesters\.org/cpan/report/ae130636\-6c13\-1014\-9537\-e9ae69de26d0

https://rt.cpan.org/Ticket/Display.html?id=95656


\-\- 
Alexandr Ciornii\, http&#8203;://chorny\.net

**Comment by p5pRT** [comment]
### From @salva

On 05/22/2014 03&#8203;:58 PM\, Alexandr Ciornii wrote&#8203;:
> \# New Ticket Created by  Alexandr Ciornii
> \# Please include the string&#8203;:  \[perl \#121944\]
> \# in the subject line of all future correspondence about this issue\.
> \# \<URL&#8203;: https://rt-archive.perl.org/perl5/Ticket/Display.html?id=121944 >
>
>
> threaded perls only\, any OS
>
> Packed\.xs&#8203;: In function ‘shuffle\_packed’&#8203;:
> Packed\.xs&#8203;:216&#8203;:32&#8203;: error&#8203;: ‘my\_perl’ undeclared \(first use in this function\)
> Packed\.xs&#8203;:216&#8203;:32&#8203;: note&#8203;: each undeclared identifier is reported only once
> for each function it appears in
>
> http&#8203;://www\.cpantesters\.org/cpan/report/d0d3fec2\-de62\-11e3\-be5c\-be9c6dfca90a
> http&#8203;://www\.cpantesters\.org/cpan/report/ae130636\-6c13\-1014\-9537\-e9ae69de26d0
>
> https://rt.cpan.org/Ticket/Display.html?id=95656


Drand01 now requires the perl context to be around\.

I have just released a new version of the module fixing it\.

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
@rjbs - Status changed from 'open' to 'resolved'
